src/sql/workbench/browser/designer/designer.ts

```diff
@@ -529,6 +529,17 @@ export class Designer extends Disposable implements IThemable {
 		return uiComponents;
 	}
 
+	// Shows the description for a property in the properties pane
```

Comment:
I don't think this method is necessary, the showDescription doesn't need to be exposed, it is implied. if the description is not empty show it, otherwise don't show it.

Reply:
that's what this is doing? i don't get your question

Reply:
this is for the scenario in case a property doesn't have a description but the element is focused, so we need to remove the previously selected description